packages/webapp-spa-server/index.js

```diff
@@ -17,11 +17,6 @@ const logger = boomerangLogger.logger;
  */
 
 function createBoomerangServer({
-  corsConfig = {
```

Comment:
hey @isakawa-carbon so I would like to keep the option to pass the cors config as an argument to the server. So I think we should keep this and use the value is truthy, otherwise default to trying to read the env var. I think something like this would work. What do you think?

```
const appCorsConfig = corsConfig || parseJSONString(CORS_CONFIG);
```

Reply:
Hey @timrbula ! Just followed your suggestion and published a new version.